src/usr/bin/gce-nic-naming

```diff
@@ -21,8 +21,8 @@ readonly LOG_TAG=${LOG_TAG:-"gce-nic-naming_$$"}
 declare PCI_BUS_PATH='/sys/bus/pci/devices'
 declare SYS_PREPEND_PATH='/sys'
 # 0x15b3:0x101e is the vendor and device ID for Mellanox CX7
-
-readonly ETHERNET_DEVICES_VENDORS=('15b3:101e')
+# 0x8086:0x145c is the vendor and device ID for Intel IDPF
```

Comment:
Can you some text that this is the PF device ID specifically?

Reply:
Done.